coalib/output/ConfigConverter.py

```diff
@@ -0,0 +1,166 @@
+import re
+
+from coalib.output.ConfWriter import ConfWriter
+from tomlkit import dumps
+from tomlkit import table, key, document
+from tomlkit.items import Key, KeyType
+from coalib.output.TomlConfWriter import TomlConfWriter
+
+
+class ConfigConverter:
+
+    """
+    Class that contains functions for generating
+    coafile from toml sections and toml file from
+    coala sections
+    """
+
+    def __init__(self, file):
+        self.__key_delimiter = ','
+        self.file = file
+        self.document = document()
+        self.unsavable_keys = ['save', 'toml_config']
```

Comment:
When we run `coala -s`, it adds the setting `save = true` to the configuration file. Traditionally it has been removed as in ConfWriter because it will cause saving functionality to be invoked every time coala is run.
Hence TomlConfWriter also has this.  So I have added it here in ConfigConverter as well.